src/main/java/com/redhat/devtools/intellij/lsp4ij/operations/hover/LSPTextHover.java

```diff
@@ -111,6 +111,9 @@ public String getQuickNavigateInfo(PsiElement element, PsiElement originalElemen
     @Nullable
     @Override
     public String generateDoc(PsiElement element, @Nullable PsiElement originalElement) {
+        if (originalElement == null || !Objects.equals(element.getContainingFile(), originalElement.getContainingFile())) {
```

Comment:
I think the proper fix is to work only with `originalElement`  (the hovered Psi element) and not with `element` (which can be a Psi Element which I suppose it can be a Psi reference (like property).